### PR TITLE
ENGDOCS-2095

### PR DESCRIPTION
**Pull request by aevesdocker**
## Description

Hardened Docker Desktop overview page was looking a lil long. This PR attempts to reduce it. No new content, just rejigging 

## Reviews

- [ ] Editorial review
- [ ] Product review

### CODE REVIEW

File: content/desktop/hardened-desktop/_index.md [hunk 1]
```diff
@@ -31,38 +31,23 @@ grid:
...
-It lets admins define and enforce robust security settings. It guarantees that developers and the containers they deploy are unable to intentionally or unintentionally circumvent these settings. Additionally, you can enhance container isolation, which helps mitigate potential security threats such as malicious payloads breaching the Docker Desktop Linux VM and the underlying host.
-
-Hardened Docker Desktop moves the ownership boundary for Docker Desktop configuration to the organization, meaning that any security controls admins set cannot be altered by the user of Docker Desktop.
-
-### Who is it for?
+Hardened Docker Desktop moves the ownership boundary for Docker Desktop configuration to the organization, meaning that any security controls administrators set cannot be altered by the user of Docker Desktop. 
 
 It is for security conscious organizations who:
-- Don’t give their users root or admin access on their machines
+- Don’t give their users root or administrators access on their machines
```
Comment (stephaurelio) [inline content/desktop/hardened-desktop/_index.md:41]: Possible typo?
```suggestion
- Don’t give their users root or administrator access on their machines
```

File: content/desktop/hardened-desktop/_index.md [hunk 2]
```diff
@@ -31,38 +31,23 @@ grid:
...
- - **Lateral movement:** Air gapped containers allows admins to configure network access restrictions for containers, thereby preventing malicious containers from performing lateral movement within the organization's network.
- - **Insider threats:** Settings Management configures and locks various Docker Desktop settings, such as proxy settings, ECI, and prevents exposure of the Docker API. This helps admins enforce company policies and prevents developers from introducing insecure configurations, intentionally or unintentionally.
+ - Malware and supply chain attacks: Registry Access Management and Image Access Management prevent developers from accessing certain container registries and image types, significantly lowering the risk of malicious payloads. Additionally, ECI restricts the impact of containers with malicious payloads by running them without root privileges inside a Linux user namespace.
+ - Lateral movement: Air gapped containers lets administrators to configure network access restrictions for containers, thereby preventing malicious containers from performing lateral movement within the organization's network.
```
Comment (stephaurelio) [inline content/desktop/hardened-desktop/_index.md:50]: I've seen this without the hyphen as well but just a suggestion: 
```suggestion
 - Lateral movement: Air-gapped containers lets administrators to configure network access restrictions for containers, thereby preventing malicious containers from performing lateral movement within the organization's network.
```

**Review by stephaurelio — APPROVED** [review]
Nice ✨ left a few tiny nits.

File: content/desktop/hardened-desktop/_index.md (hunk 2 above)
Comment (dvdksn) [reply content/desktop/hardened-desktop/_index.md:50]:
```suggestion
- Lateral movement: Air-gapped containers lets administrators configure network access restrictions for containers, thereby preventing malicious containers from performing lateral movement within the organization's network.
```

File: content/desktop/hardened-desktop/_index.md [hunk 3]
```diff
@@ -31,38 +31,23 @@ grid:
...
-
-It includes:
-
-- Settings Management, which helps admins to confidently manage and control the usage of Docker Desktop within their organization.
-- Enhanced Container Isolation (ECI), a setting that instantly enhances security by preventing containers from running as root in Docker Desktop’s Linux VM and ensures that any configurations set using Settings Management cannot be bypassed or modified by containers.
-- Registry Access Management (RAM), which allows admins to control the registries developers can access.
-- Image Access Management (IAM), which gives admins control over which images developers can pull from Docker Hub.
-- Air-gapped containers, which restricts containers from accessing unwanted network resources.
-
 ### How does it help my organisation?
```
Comment (stephaurelio) [inline content/desktop/hardened-desktop/_index.md:45]: Ah missed this as it wasn't in the diff: 
```suggestion
### How does it help my organization?
```

**Review by stephaurelio — COMMENTED** [review]
Noticed one tiny thing I missed originally 🔬

**Review by dvdksn — APPROVED** [review]

**Pull request merged**